### PR TITLE
Community features

### LINKED ISSUE #37: Create Community configuration
We need to create boolean configurations for the community. On the new dashboard community admins can choose what features will be enabled on their community. The designs can be seen here: https://www.figma.com/file/nEPG6Dkns8hDVCudVUTEzc/Dashboard?node-id=1046%3A22

We need to build only a few configurations for now, the are all toggles:

- shop
- objectives & actions
- show community stats

The steps to accomplish this will be roughly:

- Create a new migration that stores those toggles in the database (please lets discuss what's the best strategy before implementing)
- Make sure the migration fill in default values for that table on existing communities (table shouldn't allow NULL values)
- Add Ecto fields corresponding to the new table fields
- Add Absinthe fields corresponding to the Ecto fields (make sure to design NULLs properly)

### PR DESCRIPTION
**Pull request by RodrigoLeiteF**
## What issue does this PR close
Closes #37

## Changes Proposed ( a list of new changes introduced by this PR)
Adds the ability for community owners to modify their community's looks, behavior and available features.

## How to test ( a list of instructions on how to test this PR)
Run `mix test`

### CODE REVIEW

File: lib/cambiatus/commune/commune.ex [hunk 1]
```diff
@@ -474,6 +475,10 @@ defmodule Cambiatus.Commune do
   def create_community(attrs \\ %{}) do
     %Community{}
     |> Community.changeset(attrs)
+    |> Ecto.Changeset.put_assoc(:features, %{
+      actions: true,
+      shop: true
```
Comment (lucca65) [inline lib/cambiatus/commune/commune.ex:480]: default values should be handled by the database

File: test/cambiatus/commune/commune_test.exs [hunk 2]
```diff
@@ -157,5 +158,28 @@ defmodule Cambiatus.CommuneTest do
...
+      {:ok, features} =
+        Commune.update_features(community.symbol, %{
+          actions: false,
+          shop: false
+        })
+
+      assert features.community_id == community.symbol
+      assert features.shop == false
+      assert features.actions == false
+    end
```
[the remark this answers is not in the document]
Comment (RodrigoLeiteF) [reply test/cambiatus/commune/commune_test.exs:183]: Oops, I have to update the tests. Thanks! :p

File: lib/cambiatus/commune/community.ex [hunk 3]
```diff
@@ -34,6 +34,10 @@ defmodule Cambiatus.Commune.Community do
     field(:created_eos_account, :string)
     field(:created_at, :utc_datetime)
 
+    # Features
+    field(:actions, :boolean, default: true)
+    field(:shop, :boolean, default: true)
```
[the remark this answers is not in the document]
Comment (RodrigoLeiteF) [reply lib/cambiatus/commune/community.ex:39]: Good idea. What do you think @lucca65?

File: test/cambiatus/commune/commune_test.exs (hunk 2 above)
[the remark this answers is not in the document]
Comment (RodrigoLeiteF) [reply test/cambiatus/commune/commune_test.exs]: Done :+1:

**Comment by lucca65** [comment]
@RodrigoLeiteF let me know when this is ready for review

**Comment by RodrigoLeiteF** [comment]
> @RodrigoLeiteF let me know when this is ready for review

It should be ready. Your review is pending :)

**Comment by lucca65** [comment]
@RodrigoLeiteF I'll go ahead and merge this!

**Pull request merged**